### PR TITLE
edited gitignore

### PR DESCRIPTION
**Pull request by kelseyduffy**
- added the .vscode folder to the gitignore
- the -pip-wheel-metadata entry was removed by the latest update to the python gitignore template, so that's gone too

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.3 / 2021-07-20
+
+- gitignore edit
```
Comment (dcbroad3) [inline HISTORY.md:5]: Maybe `Cleaned up gitignore`? I think that's what you said in most of these PRs.

Comment (dcbroad3) [inline HISTORY.md:5]: Maybe `Cleaned up gitignore`? I think that's what you said in most of these PRs.

**Review by derekendres — APPROVED** [review]

File: HISTORY.md (hunk 1 above)
Comment (kelseyduffy) [reply HISTORY.md:5]: as discussed on the other PR, I went with 'edit' because it's a smaller change set rather than a complete rebuild of the file

Comment (kelseyduffy) [reply HISTORY.md:5]: as discussed on the other PR, I went with 'edit' because it's a smaller change set rather than a complete rebuild of the file

**Review by cbardini — APPROVED** [review]

**Pull request merged**